tests/utils/landing_state.ts

```diff
@@ -1,82 +1,41 @@
 import { sort } from "fast-sort";
 import path from "path";
-import { FileIssue, LandingState, LintAction, TaskFailure } from "tests/types";
+import { LandingState, LintAction, TaskFailure } from "tests/types";
 import { REPO_ROOT } from "tests/utils";
 
-const extractFileIssueFields = (
-  { file, line, column, message, code, level, linter, targetType, issueUrl, ..._rest }: FileIssue,
-  skip_message: boolean
-): FileIssue =>
-  skip_message
-    ? {
-        file,
-        line,
-        column,
-        // omitted message for assertions
-        code,
-        level,
-        linter,
-        targetType,
-        issueUrl,
-      }
-    : {
-        file,
-        line,
-        column,
-        message,
-        code,
-        level,
-        linter,
-        targetType,
-        issueUrl,
-      };
-
-// Also de-dupe, since sometimes we will have discrepancies in the count for multiple commands
 const extractLintActionFields = ({
-  paths,
-  linter,
-  parser,
-  report,
-  upstream,
-  fileGroupName,
-  command,
-  verb,
-  ..._rest
+  actionDurationMs: _actionDurationMs,
```

Comment:
Ideally this would just be a matcher rather than just removing it, but I was having a bit of difficulty having the matcher play nice with the `jest-specific-snapshot` dependency I was using. Will revisit if needed, but this should be fine for now.